Successfully updated:
Created Frame node vyc5dWPdn
Created Frame node phPUvysk1
Created Text node gx7S_sKog
Created Text node EOzZ01KKX
Created Frame node Qx5BCDbcR
Updated attributes for node vyc5dWPdn
Updated attributes for node phPUvysk1
Updated text for node gx7S_sKog
Updated attributes for node gx7S_sKog
Updated text for node EOzZ01KKX
Updated attributes for node EOzZ01KKX
Updated attributes for node Qx5BCDbcR
Reordered node phPUvysk1 within parent vyc5dWPdn to index 0

XML Changes:
Index: node.xml
===================================================================
--- node.xml	Before
+++ node.xml	After
@@ -3881,20 +3881,73 @@
       width="fit-content"
       height="fit-content"
       top="0px"
       left="0px"
       font="Inter"
       textTruncation="2"
   >
     Truncated Text that is long enough to be truncated
   </TruncatedTextThatIsLongEnoughToBeTruncated>
 </Frame>
 <Frame
     nodeId="yKtdVfGM5"
     position="absolute"
     width="100px"
     height="100px"
     top="0px"
     left="0px"
     <!-- background color string or project color style path (if starts with /) -->
     backgroundColor="rgb(200, 200, 200)"
  />
+<Stack
+    nodeId="vyc5dWPdn"
+    position="absolute"
+    width="400px"
+    height="300px"
+    top="0px"
+    left="0px"
+    <!-- background color string or project color style path (if starts with /) -->
+    backgroundColor="rgb(240, 240, 240)"
+    layout="stack"
+    gap="16px"
+    padding="20px"
+    stackDirection="vertical"
+    stackDistribution="center"
+    stackAlignment="center"
+>
+  <Frame
+      nodeId="phPUvysk1"
+      width="100%"
+      height="60px"
+      <!-- background color string or project color style path (if starts with /) -->
+      backgroundColor="rgb(100, 150, 200)"
+      borderRadius="8px"
+  >
+    <HeaderText
+        nodeId="gx7S_sKog"
+        position="absolute"
+        width="fit-content"
+        height="fit-content"
+        top="0px"
+        left="0px"
+        font="Inter"
+    >
+      Header Text
+    </HeaderText>
+  </Frame>
+  <BodyContentGoesHere
+      nodeId="EOzZ01KKX"
+      width="fit-content"
+      height="fit-content"
+      font="Inter"
+  >
+    Body content goes here
+  </BodyContentGoesHere>
+  <Frame
+      nodeId="Qx5BCDbcR"
+      width="100%"
+      height="1fr"
+      <!-- background color string or project color style path (if starts with /) -->
+      backgroundColor="rgb(255, 255, 255)"
+      borderRadius="4px"
+   />
+</Stack>


Note: Set zoomIntoView=false if you want to use Framer app while MCP is working.